examples/inertia_relief/inertia_relief_example.cpp

```diff
@@ -0,0 +1,542 @@
+// Copyright (c) 2019-2024, Lawrence Livermore National Security, LLC and
+// other Serac Project Developers. See the top-level LICENSE file for
+// details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+/**
+ * @file smith_example.cpp
+ *
+ * @brief Basic Smith example
+ *
+ * Intended to verify that external projects can include Smith
+ */
+
+#include "serac/physics/solid_residual.hpp"
+#include "serac/physics/functional_objective.hpp"
+
+#include "serac/infrastructure/application_manager.hpp"
+#include <serac/physics/state/state_manager.hpp>
+#include "serac/physics/materials/solid_material.hpp"
+#include "serac/physics/mesh.hpp"
+#include "serac/physics/common.hpp"
+#include "mfem.hpp"
+#include "serac/physics/tests/physics_test_utils.hpp"
+#include "serac/numerics/functional/tensor.hpp"
+#include "serac/physics/scalar_objective.hpp"
+
+#include "problems/Problems.hpp"
+#include "solvers/HomotopySolver.hpp"
+#include "utilities.hpp"
+
+#include <axom/sidre.hpp>
```

Comment:
```suggestion
#include "serac/serac.hpp"

#include "mfem.hpp"

// ContinuationSolver headers
#include "problems/Problems.hpp"
#include "solvers/HomotopySolver.hpp"
#include "utilities.hpp"

#include "axom/sidre.hpp"
```

Reply:
I recommend in the Conuation project to move your headers into a subdirectory so they do not clash with other projects. `utilities.hpp` for example is very generically named.

Reply:
I agree. I'll give it a try.